compiler/rustc_error_codes/src/error_codes/E0547.md

```diff
@@ -0,0 +1,37 @@
+The `issue` value is missing in a stability attribute.
```

Comment:
Would this be clearer if it is flipped? I wonder if the term is called "value"?
```suggestion
A stability attribute is missing `issue`.
```

Reply:
Honestly, I can't see any benefit. Anyway, it's not a big deal for me :).

@GuillaumeGomez , what do you think?

Reply:
I find the suggested one more difficult to understand.